src/realm/table.cpp

```diff
@@ -1765,6 +1794,12 @@ bool Table::is_list(ColKey col_key) const
     return col_key.get_attrs().test(col_attr_List);
 }
 
+bool Table::is_set(ColKey col_key) const
```

Comment:
I would rather avoid this function. We should use ColKey::is_set()

Reply:
I added it for completeness, considering `Table::is_list()`, but we should probably move away from that, as you say. I will remove this.